packages/next/lib/install-dependencies.ts

```diff
@@ -0,0 +1,37 @@
+import chalk from 'chalk'
+import path from 'path'
+
+import { MissingDependency } from './has-necessary-dependencies'
+import { shouldUseYarn } from './helpers/should-use-yarn'
+import { install } from './helpers/install'
+import { getOnline } from './helpers/get-online'
+
+export type Dependencies = {
+  resolved: Map<string, string>
+}
+
+export async function installDependencies(
+  baseDir: string,
+  deps: any,
+  dev: boolean = false
+) {
+  const useYarn = shouldUseYarn()
+  const isOnline = !useYarn || (await getOnline())
+
+  if (deps.length) {
+    console.log()
+    console.log(`Installing package${deps.length > 1 ? 's' : ''}:`)
```

Comment:
Should this say "Installing devDependencies" instead?

Reply:
Updated to say either `devDependencies` or `dependencies` :)